client/package.json

```diff
@@ -49,6 +51,7 @@
     "@nivo/line": "^0.72.0",
     "@nivo/pie": "^0.72.0",
     "@nivo/treemap": "^0.72.0",
+    "axe-core": "^4.3.1",
```

Comment:
Showing up in both dependencies and dev-dependencies. Just manually delete it from regular dependencies (which are packages that are used by the run-time client).